NavigationMenu.md

```diff
@@ -46,9 +46,13 @@
   * [Information extraction](./guides/V1_wwsg_InformationExtraction.md)
   * [Information extraction annotations](./guides/V1_Annotation_Message_Information_Extraction.md)
 
+* ## [Actions and Events](./guides/actions_and_events_overview.md)
+  * [Handling Client Actions](./guides/handling_client_actions.md)
+   * [App actions callback](./guides/App_action_Callback.md)
+   * [Get action context information](./guides/get_action_context.md)
+  
 * ## [Apps](./guides/V1_Apps.md)
   * [Prepare your App to run](./guides/V1_PreparingYourApp.md)
-  * [Make your App configurable](./guides/V1_MakeAppsConfigurable.md)
```

Comment:
we can't remove this one yet - for two reasons

- there are links to it from the **Your Apps** area
- we need to continue with this section for all of those apps that have already been set up using this pattern - we do need to add the _deprecated_ moniker here

Reply:
Added this section back and added (Deprecated) in the title. Thanks!